scanner/src/main/kotlin/provenance/PostgresPackageProvenanceStorage.kt

```diff
@@ -86,6 +86,13 @@ class PostgresPackageProvenanceStorage(
             }.map { it[table.result] }.firstOrNull()
         }
 
+    override fun readProvenances(id: Identifier): List<PackageProvenanceResolutionResult> =
+        database.transaction {
+            table.select {
+                table.identifier eq id.toCoordinates()
+            }.map { it[table.result] }
```

Comment:
Nit: This could be a one-liner (or probably *both* `select` and ` map` should be wrapped).

Reply:
I have chosen this formatting because it matches the formatting of the other read functions.